src/lib/core/data-table/data-table.ts

```diff
@@ -62,14 +63,17 @@ export class CdkTable implements CollectionViewer {
   @Input() dataSource: DataSource<any>;
 
   // TODO(andrewseguin): Remove max value as the end index
-  // and instead calculate the view on init and scroll.
+  //   and instead calculate the view on init and scroll.
   /**
    * Stream containing the latest information on what rows are being displayed on screen.
    * Can be used by the data source to as a heuristic of what data should be provided.
    */
-  viewChanged =
+  viewChange =
       new BehaviorSubject<{start: number, end: number}>({start: 0, end: Number.MAX_VALUE});
 
+  /** Stream that emits when a row def has a change to its array of columns to render. */
+  columnsChange = new Observable<void[]>();
```

Comment:
I agree but it makes Typescript happy. The merged streams are each Observable<void> and combined make up Observable<void[]>